### PR TITLE
feat:  backend setup (python and django)

### PR DESCRIPTION
**Pull request by DManuel010**
- configured python3 setup
- configured django setup
- use virutalenv and requirements.txt to setup your own python3 virtual environment + dependencies

### CODE REVIEW

**Comment by bdatdo0601** [comment]
I think it's a good idea to move these in a separate directory like `django-backend`

**Review by simpsona4 — APPROVED** [review]
Self-Note: Beginnings of user auth, and other backend functionalities.

**Review by bdatdo0601 — CHANGES_REQUESTED** [review]
We also going to need a readme documentation on how to set it up

File: requirements.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+django==3.0.2
+django-mongodb-engine==0.6.0
+djangotoolbox==1.8.0
+pylint==2.4.4
+pymongo==3.10.1
```
Comment (bdatdo0601) [inline requirements.txt]: This should sit within `django-backend` file since this is only related to Python

**Review by bdatdo0601 — CHANGES_REQUESTED** [review]
Looks good to start! But you probably need to rebase with dev

**Comment by bdatdo0601** [comment]
[Python and DJango setup](https://trello.com/c/4wVuwVAB/50-python-and-django-setup)

**Review by bdatdo0601 — APPROVED** [review]
LGTM!

**Pull request merged**